content_schemas/examples/content_block_email_address/publisher_v2/example.json

```diff
@@ -0,0 +1,11 @@
+{
+  "locale": "en",
+  "schema_name": "content_block_email_address",
+  "document_type": "content_block_email_address",
+  "title": "Government Digital Service",
+  "description": "Government Digital Service",
```

Comment:
What's description being used for here? And `title` I suppose, unless that's supposed to represent the name of the thing that has an email address?

Reply:
I've updated this to be clearer. It's still very much in flux though